todo/src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
```

Comment:
This is the first time I've seen local storage implemented along w/ Redux. Great job reading the docs and implementing it.